cleverpush/src/main/java/com/cleverpush/listener/NotificationOpenedListener.java

```diff
@@ -1,7 +1,10 @@
 package com.cleverpush.listener;
 
+import android.app.Activity;
+
 import com.cleverpush.NotificationOpenedResult;
 
 public interface NotificationOpenedListener {
-    void notificationOpened(NotificationOpenedResult result);
+    void notificationOpened(NotificationOpenedResult result, Activity finishActivity);
```

Comment:
revert

Reply:
It's needed for finishing the actual activity - Check the above comment 
cc: @cpKevinrodriguesnobre